mantle/cmd/kola/testiso.go

```diff
@@ -592,16 +607,24 @@ func awaitCompletion(ctx context.Context, inst *platform.QemuInstance, outdir st
 
 func printResult(mode string, duration time.Duration, err error) bool {
```

Comment:
I think we can simplify this to just print the test name now that it contains all the information about the combination tested. This makes it more similar to the `kola run` output.